### PR TITLE
Adding orb_slam2_with_maps-odom to documentation index for distro

### PR DESCRIPTION
**Pull request by SuperAruba**
Adding orb_slam2_with_maps-odom to documentation index for distro.

### CODE REVIEW

File: kinetic/distribution.yaml [hunk 1]
```diff
@@ -9514,6 +9514,16 @@ repositories:
       url: https://github.com/appliedAI-Initiative/orb_slam_2_ros.git
       version: master
     status: maintained
+  orb_slam2_with_maps-odom:
```
Comment (ivanpauno) [inline kinetic/distribution.yaml:9517]: From the [reviewing guidelines](https://github.com/ros/rosdistro/blob/master/REVIEW_GUIDELINES.md):

> The source repository should generally contain one or more ROS packages (ROS packages are ones that have a package.xml and use either catkin or ament build helpers). Packages that are not ROS packages can be accepted, but they should be rare and require special handling in the release repository.

I don't think that the repository satisfies that requirement, as there aren't any ros packages in it.

**Review by ivanpauno — CHANGES_REQUESTED** [review]

File: kinetic/distribution.yaml (hunk 1 above)
Comment (tfoote) [reply kinetic/distribution.yaml:9517]: Yeah, without ROS packages in it we cannot index it or generate documentation. I'm going to close this.

**Review by tfoote — CHANGES_REQUESTED** [review]

File: kinetic/distribution.yaml (hunk 1 above)
Comment (SuperAruba) [reply kinetic/distribution.yaml:9517]: I have changed repository.

**Comment by ivanpauno** [comment]
@SuperAruba the repository seems to be a ros package now, so it can be added.
The repository is still missing a `LICENSE.md` file.

Reopening for more visibility.

**Comment by ivanpauno** [comment]
Oh, there's a new PR so I cannot reopen this one: https://github.com/ros/rosdistro/pull/26594/.

**Pull request closed without merging**